docs/install/install-kn.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Installing the `kn` Client"
+linkTitle: "Installing the `kn` Client"
+weight: 10
+type: "docs"
+---
+
+## Install `kn` using the nightly executable binary
```

Comment:
We should put an intro sentence on the page, so that the title doesn't sit right on top of this h2.